### DISCUSSION TITLE
Trouble with the Entities import

### DISCUSSION THREAD

**Discussion by Harley-L** [AlmasB/FXGL#1132] [Q&A] [answered]
Hello,
I have been using version 11.17 of FXGL and have been having problems importing com.almasb.fxgl.entity.Entities. 

com.almasb.fxgl.entity.Entity imports fine as well as every other class as far as I can tell. Right now, I need the entities class to make screen bounds with FXGL.getGameWorld().addEntity(Entities.makescreenbounds(40)). 

Is there a way I can get around this or would you happen to know how I can fix this issue?

Thanks a ton!

**Comment by AlmasB** [comment, marked as answer]
@Harley-L 

Hi, `Entities.*` is legacy API, the current API is `FXGL.entityBuilder().*`. You will be able to find screen bounds generation there.
